--- kbd-1.14.1.20081023/src/unicode_start.unicode_start_vs_setfont	2008-10-24 01:25:22 +0400
+++ kbd-1.14.1.20081023/src/unicode_start	2008-10-24 01:27:18 +0400
@@ -65,23 +65,4 @@ fi
 if [ -t 1 -a -t 2 ]; then
 	printf '\033%%G'
 fi
 stty iutf8
-
-# Tell the graphics card how to display Unicode characters not
-# contained in the IBM 437 character set (on PCs). The font should
-# have a Unicode map attached, or explicitly specified, e.g.,
-# by giving `def.uni' as a second argument.
-
-case "$#" in
-	2)
-		setfont "$1" -u "$2"
-		;;
-	1)
-		setfont "$1"
-		;;
-	0)
-		;;
-	*)
-		echo "usage: unicode_start [font [unicode map]]"
-		;;
-esac
